sentry-android-core/src/main/java/io/sentry/android/core/ManifestMetadataReader.java

```diff
@@ -78,6 +78,8 @@ final class ManifestMetadataReader {
   static final String CLIENT_REPORTS_ENABLE = "io.sentry.send-client-reports";
   static final String COLLECT_ADDITIONAL_CONTEXT = "io.sentry.additional-context";
 
+  static final String PERFORM_ACTIVITY_FRAMES_TRACKING = "io.sentry.activity-frames-tracking";
```

Comment:
we already have a pattern, like `io.sentry.traces.activity.enable`, should we also follow it here and change it to smth like `io.sentry.traces.activity.frames-tracking` ?